hadoop-ozone/dist/src/main/compose/ozone-topology/test.sh

```diff
@@ -33,6 +33,20 @@ execute_robot_test scm basic/basic.robot
 
 execute_robot_test scm topology/scmcli.robot
 
+# Ensure data can be read even when a full rack
+# is stopped.
+execute_robot_test scm topology/loaddata.robot
+
+stop_containers datanode_1 datanode_2 datanode_3
+
+execute_robot_test scm topology/readdata.robot
+
+start_containers datanode_1 datanode_2 datanode_3
+
+stop_containers datanode_4 datanode_5 datanode_6
+
```

Comment:
what about if datanode_1, datanode_2 and datnaode_3 are not yet started / registered but 4-6 are stopped. Do we need some safety check here?

In the `testlib.sh/start_docker_env` we have a `wait_for_safemode_exit` part. We might need it here.

Reply:
Looking at some of the intermittent failures, I think we do need a safety check, but the safemode check will not do it. I believe we need to wait for the xceiver  port on the DNs to come up. I have a plan for this which I will post shortly.

However we will still need HDDS-3107 as that is another reason this test can fail.